### PR TITLE
[OM-93337]: update kubeturbo yaml to pull image from icr.io

### PR DESCRIPTION
**Pull request by libai98**
pull image from icr.io.

➜  ~ skopeo list-tags docker://icr.io/cpopen/turbonomic/kubeturbo
{
    "Repository": "icr.io/cpopen/turbonomic/kubeturbo",
    "Tags": [
        "8.6.3",
        "8.6.4",
        "8.6.5",
        "8.6.6",
        "8.7.0",
        "8.7.1",
        "8.7.2-SNAPSHOT",
        "8.7.2",
        "8.7.3-SNAPSHOT",
        "8.7.3",
        "8.7.4-SNAPSHOT"
    ]
}
➜  ~ docker pull icr.io/cpopen/turbonomic/kubeturbo:8.7.3
8.7.3: Pulling from cpopen/turbonomic/kubeturbo
889173b88889: Pull complete
4f4fb700ef54: Pull complete
0d03d79a56d6: Pull complete
1f40e47ccbf8: Pull complete
7000d1a4a06b: Pull complete
0732c70eb5d9: Pull complete
6d420c6b9438: Pull complete
7b817dfffdb2: Pull complete
7bedf40435f3: Pull complete
Digest: sha256:502a19b191b2b3d8437e72a4c4cf2d19d140b3ea00e6974201e39e84172cb24f
Status: Downloaded newer image for icr.io/cpopen/turbonomic/kubeturbo:8.7.3

### CODE REVIEW

**Comment by ading1977** [comment]
@libai98 , we need to update the helm charts and operator too.

**Comment by ading1977** [comment]
Shall we change the default tag in all places to at least `8.7.3`?

File: deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml [hunk 1]
```diff
@@ -33,8 +33,8 @@ spec:
 
   # Uncomment next lines to specify a repository and image tag for kubeturbo
   #image:
-  #  repository: registry.connect.redhat.com
-  #  tag: 8.4.0
+  #  repository: icr.io/cpopen/turbonomic/kubeturbo
+  #  tag: 8.7.3
   # Uncomment to use an image from RHCC for cpu-frequency getter job - predefined in OCP Operator Hub version
   #  busyboxRepository: registry.access.redhat.com/ubi8/ubi-minimal
```
Comment (ading1977) [inline deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml:39]: What is our default `busyboxRepository` now?

File: deploy/kubeturbo-operator/helm-charts/kubeturbo/values.yaml [hunk 2]
```diff
@@ -6,8 +6,8 @@ replicaCount: 1
 
 # Replace the image with desired version
 image:
-  repository: turbonomic/kubeturbo
-  tag: 8.6.1
+  repository: icr.io/cpopen/turbonomic/kubeturbo
+  tag: 8.7.3
   pullPolicy: IfNotPresent
 #  busyboxRepository: busybox
```
Comment (ading1977) [inline deploy/kubeturbo-operator/helm-charts/kubeturbo/values.yaml:12]: Again, can we double check the `busybox` image location? Is it moved to `icr.io` as well? What should be the default values?

File: deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml (hunk 1 above)
Comment (pankajbatra1) [reply deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml:39]: -->$skopeo list-tags docker://icr.io/cpopen/turbonomic/busyboxturbo
{
    "Repository": "icr.io/cpopen/turbonomic/busyboxturbo",
    "Tags": [
        "1.0"
    ]
}

**Comment by ading1977** [comment]
Please connect with @SumanthKodali999 to let him know that `kubeturbo` **crd** is changed, and he should update them when working on https://vmturbo.atlassian.net/browse/OM-93289 for certified operator.

**Review by pankajbatra1 — APPROVED** [review]

File: deploy/kubeturbo-operator/helm-charts/kubeturbo/values.yaml (hunk 2 above)
Comment (libai98) [reply deploy/kubeturbo-operator/helm-charts/kubeturbo/values.yaml:12]: Busybox work is postponed after discussion with David Orsie and Billy.

File: deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml (hunk 1 above)
Comment (libai98) [reply deploy/kubeturbo-operator/deploy/crds/charts_v1alpha1_kubeturbo_cr.yaml:39]: Busybox work is postponed after discussion.

**Pull request merged**